diagnostic.md

```diff
@@ -6,70 +6,76 @@ Place your responses inside the fenced code-blocks where indicated by comments.
 What are two ways of telling rails where to route a GET request to '/books/4'?
 
 ```rb
-// your response here
+get '/books/:id' to: 'books#show'
+
+resources :books, only: [:show]
+
+
 ```
 
 In what file does the above code go?
 
 ```md
-// your response here
+config/routes.rb
 ```
 
 In your own words, what does the command `bundle exec rake db:migrate` do?
 
 ```md
-// your response here
+It executes the rake script to run db migrations that haven't ran yet and are queued.
 ```
 
 What is the difference between `rails g scaffold things` and
 `rails g controller things`?
 
 ```md
-// your response here
+g scaffold sets up a model, db migration, controller, view, and test suite.
+
+g controller would just generate a controller.
 ```
 
 What is the job of the `serializer`?
 
 ```md
-// your response here
+creating custom json
 ```
 
 What do we expect the command `Patient.all` to return?
 
 ```md
-// your response here
+all patients
 ```
 
 We use the constant `Patient` in the `PatientsController`, but where is it
 actually defined?
 
 ```md
-// your response here
+patient_spec.rb
 ```
 
 Why do our `model` files appear to be lacking so much code, when they're doing
 so much for us?
 
 ```md
-// your response here
+They inherit from Application Record so they many things from it you don't see in the model.
 ```
 
 What does typing `bin/rails dbconsole` in a directory that houses a rails app do for
 us?
 
 ```md
-// your response here
+Opens the console for the database for that app.
```

Comment:
This command will allow you to interact with Rails app, whereas the next one will let you interact with the underlying database